packages/components/components/elvis-breadcrumb/src/react/elvia-breadcrumb.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import percySnapshot from '@percy/playwright';
+import { expect, test } from '@playwright/experimental-ct-react';
+import React from 'react';
+
+import { Breadcrumb, type BreadcrumbProps } from '../../react';
+
+const items: BreadcrumbProps['items'] = [
+  { href: '/', text: 'Home' },
+  { href: '/catalog', text: 'Catalog' },
+  { href: '/catalog/product', text: 'Product' },
+];
+
+test('should render', async ({ mount }) => {
+  const component = await mount(<Breadcrumb items={items} />);
+  await expect(component).toBeAttached();
+});
+
+test.describe('breadcrumb screenshots desktop', () => {
+  test.use({ viewport: { width: 800, height: 250 } });
+
+  test('breadcrumb screenshots desktop', async ({ mount, page }) => {
+    await mount(<Breadcrumb items={items} />);
+    await percySnapshot(page, 'Breadcrumb Desktop');
+  });
+});
```

Comment:
Would it be possible to add a test that checks the hover state on a link to get the green underline?